sites/partners/src/components/applications/PaperApplicationForm/MultiselectQuestionsMap.tsx

```diff
@@ -0,0 +1,189 @@
+import React, { useEffect, useState } from "react"
```

Comment:
There is a lot of duplication between this file and `BuildingDetails.tsx`. I wonder if there are chunks of it related to the map and lat/long api call that we could pull out to a separate file. Maybe we should do that as part of a different ticket though